### ISSUE TITLE
Remove '/' requirement for root (SSH under Windows)

### ISSUE THREAD

**Issue by garrowson** [SchoofsKelvin/vscode-sshfs#129]
I use the Windows SSH Client / Server feature and have problems configuring a workspace cause the config requires a forward slash in front of the path.

**Comment by garrowson** [comment]
A way to remove/ignore those warnings would also be nice
![image](https://user-images.githubusercontent.com/49206998/56083657-3c025780-5e28-11e9-9945-67a1c67b8fdf.png)

**Comment by SchoofsKelvin** [comment]
I'm not sure how well that'll work. Since it seems to be a "proper" SSH system with just specially named directories, you can try `/D:`, or otherwise edit settings.json directly.

**Comment by garrowson** [comment]
No, tried before. Gives warning in config and won't change directory

**Comment by SchoofsKelvin** [comment]
I've quickly installed the OpenSSH Server feature, started the service, and setup a quick ssh fs config for it:
```jsonc
    {
      "name": "issue-129",
      "password": true,
      "putty": "Localhost", // PuTTY session just specifies my username and "localhost" as host
      "root": "/D:Lenovo"
    }
```
I tried both no root, `/D:`, `/D:/` `/D:Lenovo` and `/D:/Lenovo` as root, and all worked fine for me. When using no root, I also had both my `C:` and `D:`, like in your screenshot.

I did get that warning exclamation mark, but after closing the workspace and reconnecting, it didn't appear. I remember from previous bug/tests that that exclamation mark basically never disappears in the current window.
